model_bakery/utils.py

```diff
@@ -46,4 +46,4 @@ def seq(value, increment_by=1, start=None):
                 yield series_date
     else:
         for n in itertools.count(start or increment_by, increment_by):
-            yield value + type(value)(n)
+            yield value + type(value)(n) + (suffix or type(value)())
```

Comment:
I have double feelings about the ambiguity between `start` and `suffix` kwargs, but I don't see how can we escape from that. IMHO, the suffix parameter should only be available for text fields... What do you think about this?

Reply:
What do you think about something like this:

```python
if suffix and not type(suffix) is str:
    raise TypeError("Sequences suffix can only be a string")
for n in itertools.count(start or increment_by, increment_by):
    value_type = type(value)
    if suffix and not value_type is str:
        raise TypeError("Sequences with suffix can only be used with text values")
```

Reply:
I like that approach. I was hesitant to add that much boilerplate at first, however, I think it makes more sense to prevent accidental bad usage. I will update the PR.